airbyte-ci/connectors/metadata_service/lib/metadata_service/models/src/ConnectorMetrics.yaml

```diff
@@ -0,0 +1,30 @@
+---
+"$schema": http://json-schema.org/draft-07/schema#
+"$id": https://github.com/airbytehq/airbyte/airbyte-ci/connectors_ci/metadata_service/lib/models/src/ConnectorMetrics.yaml
+title: ConnectorMetrics
+description: Information about the source file that generated the registry entry
+type: object
+properties:
+  all:
+    type: ConnectorMetric
+  cloud:
+    type: ConnectorMetric
+  oss:
+    type: ConnectorMetric
+definitions:
+  ConnectorMetric:
+    type: object
+    properties:
+      usage:
+        oneOf:
+          - type: string
+          - type: string
+            enum: [low, medium, high]
+      sync_success_rate:
+        oneOf:
+          - type: string
+          - type: string
+            enum: [low, medium, high]
```

Comment:
I'm not sure why you're using a `oneOf` here instead of a "pure" enum.

Reply:
I did this to allow for unexpected values outside of the enum.

e.g. if the data team decides to add `med-high`, `near-zero`, or `HIGH` as an option we wont break.